### PR TITLE
Added shows list

### PR DESCRIPTION
**Pull request by will-hedges**
Formatted similarly to Matt's list

### CODE REVIEW

**Review by loganallengray — APPROVED** [review]
👍

**Review by yerwhile — CHANGES_REQUESTED** [review]
show objects do not have genre property (genreId)

File: scripts/shows.js [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+    // need to take the foreign key of the show actorId
+    //  and match to the actor id primary key
+    let starActor = null;
+    for (const actor of actors) {
+      if (show.actorId === actor.id) {
+        starActor = actor;
+      }
+    }
+    // build and add a list item string to the html string
+    html += `<li>${show.name} (${show.genre}) starring ${starActor}</li>`;
```
Comment (will-hedges) [inline scripts/shows.js:19]: need to access correct property

Comment (will-hedges) [reply scripts/shows.js]: done

**Review by ristopherCh — APPROVED** [review]

**Review by yerwhile — APPROVED** [review]
I have approved the absolute idea of changes in this module.

**Pull request merged**